src/crypto/random.js

```diff
@@ -41,6 +41,8 @@ export default {
     const buf = new Uint8Array(length);
     if (typeof window !== 'undefined' && window.crypto && window.crypto.getRandomValues) {
       window.crypto.getRandomValues(buf);
+    } else if (typeof crypto !== 'undefined' && crypto && crypto.getRandomValues) {
+      crypto.getRandomValues(buf);
```

Comment:
Instead of adding these two lines, could you replace the first two by them? (I.e. remove the first if and just keep this one)

Reply:
If i do that, the test aren't passing anymore.

Reply:
Done :)